README.md

```diff
@@ -230,6 +230,15 @@ the source for `JsonClient`. Effectively, you extend it, and set the
 appropriate options in the constructor and implement a `write` (for
 put/post) and `parse` method (for all HTTP bodies), and that's it.
 
+If the server responds with a `content-md5` header, `StringClient` will
+automatically verify the response body md5 hash, and return a BadDigest error
```

Comment:
This is a breaking change -- since now we're throwing an Error in a case where we didn't before. What I would suggest is that instead of making this the default, make it optional, and thus a minor version bump -- we can enable this in the next major version.

Reply:
No - I have not changed this handling - that behaviour already exists (I just documented it).

The restify client calculates the md5 hash when a content-md5 response header is available - and it returns a BadDigest error (via the callback) when it doesn't match.